NuGet.config

```diff
@@ -12,6 +12,7 @@
     <!-- Microsoft.CodeAnalysis.Test.Resource.Proprietary -->
     <add key="roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
     <add key="roslyn-analyzers" value="https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json" />
+    <add key="dotnet-format" value="https://dotnet.myget.org/F/format/api/v3/index.json" />
```

Comment:
This is no longer necessary as the referenced version of dotnet-format is available from nuget.

Reply:
Removed this change.